Fixes #5729 - Update Shobdon (EGBS) SMR

CLC0609 opened the pull request:
Fixes #5729 

# Summary of changes
Added new intermediate hold A2
Renamed holding point A to A1

# Screenshots (if necessary)
![image](https://github.com/user-attachments/assets/f6a54296-05f4-4877-b82c-446dcac93346)

This is my first time editing an SMR so I might have done something wrong :)

PLM1995 commented on Airports/EGBS/SMR/Labels.txt:
```diff
@@ -1,5 +1,5 @@
 ;Holding Points
-"A"  N052.14.34.118 W002.52.31.171 smrStandHold
+"A1"  N052.14.34.118 W002.52.31.171 smrStandHold
 "B1" N052.14.30.164 W002.53.20.859 smrStandHold
```
```suggestion
"A1" N052.14.34.118 W002.52.31.171 smrStandHold
"B1" N052.14.30.164 W002.53.20.859 smrStandHold
```
Alignment.

PLM1995 commented on Airports/EGBS/SMR/Labels.txt:
Should there be an A2 label?

PLM1995 reviewed (commented):
Just FYI, it is usual to provide a screenshot for SMR changes to allow for speedy review :)

CLC0609 commented:
> Just FYI, it is usual to provide a screenshot for SMR changes to allow for speedy review :)

Ok thanks for letting me know, just added one now,

I've also added the A2 label.
